### DISCUSSION TITLE
Troubleshooting

### DISCUSSION THREAD

**Discussion by hajfly** [RaspAP/raspap-webgui#1495] [Q&A] [answered]
Hello, help.... I have been trying for several days to install raspap on a Raspberry Pi4 and set up a Wifi access point, unfortunately only with limited success.
Originally the whole thing worked once, but I have now reset the Pi and reinstalled Raspap. After starting, the network appears briefly in my network list as intended, but then disappears again. It is not possible to connect, not even via the QR code in the user interface. According to the user interface, everything works.... Restarting does not work either. 
I have pi-hole running on another Pi, but even entering the PiHole as a DNS upstream server is not successful.
That's as far as my skills go and I'm at the end of my tether. I have searched for similar error messages, but have already failed in part to understand or implement the answers...
Maybe someone is after a good deed and can help with....

**Comment by billz** [comment]
RaspAP has a number of tools to help you perform a self-diagnosis when things behave unexpectedly. Have you looked at the docs? Read the FAQ? Here's a short list, in order of priority:

1. Generate a debug log
2. Enable logging for hostapd and DHCP
3. Perform a system reset 

Without systems logs or other details, there's nothing I (or anyone else) can do to assist you with troubleshooting.

**Comment by hajfly** [comment]
Here we go:

[raspap_debug-2.log](https://github.com/RaspAP/raspap-webgui/files/13902720/raspap_debug-2.log)

DHCP:

```
[snip]
Jan 11 12:56:05 dnsmasq[2625]: exiting on receipt of SIGTERM
Jan 11 12:56:10 dnsmasq[3025]: started, version 2.89 cachesize 150
Jan 11 12:56:10 dnsmasq[3025]: compile time options: IPv6 GNU-getopt DBus no-UBus i18n IDN2 DHCP DHCPv6 no-Lua TFTP conntrack ipset nftset auth cryptohash DNSSEC loop-detect inotify dumpfile
Jan 11 12:56:10 dnsmasq-dhcp[3025]: DHCP, IP range 10.3.141.50 -- 10.3.141.254, lease time 12h
Jan 11 12:56:10 dnsmasq[3025]: reading /etc/resolv.conf
Jan 11 12:56:10 dnsmasq[3025]: using nameserver 192.168.178.85#53
Jan 11 12:56:10 dnsmasq[3025]: using nameserver 2003:cd:b730:f200:509e:1c62:e56d:ff66#53
Jan 11 12:56:10 dnsmasq[3025]: read /etc/hosts - 7 names
```

Hotspot:

wlan0: interface state ENABLED->DISABLED
wlan0: AP-DISABLED 
wlan0: CTRL-EVENT-TERMINATING 
nl80211: deinit ifname=wlan0 disabled_11b_rates=0
wlan0: interface state UNINITIALIZED->COUNTRY_UPDATE
wlan0: interface state COUNTRY_UPDATE->ENABLED
wlan0: AP-ENABLED

**Comment by billz** [comment]
Your onboard wireless interface `wlan0` is being managed by wpa_supplicant (connected to muesliFRITZ!).
It is simultaneously configured as an AP. Choose one or the other, not both.

If you've used the official Raspberry Pi imager to prepare your SD card, it will ask if you want to authenticate with an existing Wi-Fi network. Skip this step.

If you're trying to configure RaspAP as a wireless repeater, attach an external USB wi-fi adapter. See the FAQ for a recommended list of adapters with in-kernel drivers.

**Comment by hajfly** [comment]
ok - this sounds reasonable indeed - will give it a try - thanks for the moment

**Reply by hajfly** [reply, marked as answer]
Once you know the error, it is very easy to fix it. Thanks - feel hugged - you made me very happy!

**Reply by billz** [reply]
Glad to hear it! Enjoy your AP

**Reply by djtp** [reply]
If, like me, you had to use a Wireless connection to install Raspap - you can use the nmtui command to just delete the preconfiguration setting from wlan0- without needing to go through the whole installation process again

**Comment by pageheller** [comment]
I had a similar problem. But, I noticed a number of different things preventing me from getting a network SSID available. My WLAN saw it as an ARP poisoning attack and a duplicate IP address. (Two blocking actions.) I looked through every device on my network, but didn't see any duplicate. I have no idea why it was blocking it based on the duplication. I had to set a rule to disregard this duplicate. Then, I had to use nmtui to delete the wlan0 I had set up with raspi-config upon installation. Now, I finally see the network raspi-netgui.
